topics/proteomics/tutorials/metaquantome-data-creation/tutorial.bib

```diff
@@ -0,0 +1,63 @@
+
+@article{Batut2018,
+  doi = {10.1016/j.cels.2018.05.012},
+  url = {https://doi.org/10.1016/j.cels.2018.05.012},
+  year = {2018},
+  month = jun,
+  publisher = {Elsevier {BV}},
+  volume = {6},
+  number = {6},
+  pages = {752--758.e1},
+  author = {B{\'{e}}r{\'{e}}nice Batut and Saskia Hiltemann and Andrea Bagnacani and Dannon Baker and Vivek Bhardwaj and
+           Clemens Blank and Anthony Bretaudeau and Loraine Brillet-Gu{\'{e}}guen and Martin {\v{C}}ech and John Chilton
+           and Dave Clements and Olivia Doppelt-Azeroual and Anika Erxleben and Mallory Ann Freeberg and Simon Gladman and
+           Youri Hoogstrate and Hans-Rudolf Hotz and Torsten Houwaart and Pratik Jagtap and Delphine Larivi{\`{e}}re and
+           Gildas Le Corguill{\'{e}} and Thomas Manke and Fabien Mareuil and Fidel Ram{\'{i}}rez and Devon Ryan and
+           Florian Christoph Sigloch and Nicola Soranzo and Joachim Wolff and Pavankumar Videm and Markus Wolfien and
+           Aisanjiang Wubuli and Dilmurat Yusuf and James Taylor and Rolf Backofen and Anton Nekrutenko and Bj\"{o}rn Gr\"{u}ning},
+  title = {Community-Driven Data Analysis Training for Biology},
+  journal = {Cell Systems}
+}
+@article{Jagtap2015,
+  doi = {10.1002/pmic.201500074},
+  url = {https://doi.org/10.1002/pmic.201500074},
+  year = {2015},
+  month = jul,
+  publisher = {Wiley},
+  volume = {15},
+  number = {20},
+  pages = {3553--3565},
+  author = {Pratik D. Jagtap and Alan Blakely and Kevin Murray and Shaun Stewart and Joel Kooren and James E. Johnson and Nelson L. Rhodus and Joel Rudney and Timothy J. Griffin},
+  title = {Metaproteomic analysis using the Galaxy framework},
+  journal = {{PROTEOMICS}}
+}
+@online{Metaproteomics community effort,
```

Comment:
```suggestion
@online{Metaproteomics_community_effort,
```

Reply:
Spaces are disrupting the references. Remove them in the names of the references here in the .bib file and also where they are referenced in the tutorial.md file.

Reply:
done..looks like the checks have passed..